bufstream/docker-compose-quickstart/docker-compose.yml

```diff
@@ -0,0 +1,79 @@
+services:
+  # We use PostgreSQL (https://www.postgresql.org/) for storing metadata for bufstream.
+  postgres:
+    # Our minimum supported version for PostgreSQL is 14.
+    image: postgres:14
+    container_name: postgres
+    environment:
+      - POSTGRES_USER=root
+      - POSTGRES_PASSWORD=examplepassword
+      - POSTGRES_DB=examplebufstreamdb
+    network_mode: host
+    healthcheck:
+      test: ["CMD", "pg_isready", "-d", "examplebufstreamdb"]
+
+  minio:
+    image: minio/minio:RELEASE.2025-05-24T17-08-30Z
+    container_name: minio
+    environment:
+      - MINIO_ROOT_USER=admin
+      - MINIO_ROOT_PASSWORD=password
+      - MINIO_DOMAIN=localhost
+    network_mode: host
+    healthcheck:
+      test: ["CMD", "curl", "--silent", "--fail", "--output", "/dev/null", "http://localhost:9000/minio/health/live"]
+    command:
+      - server
+      - /data
+      - --console-address
+      - :9001
+
+  # We use Minio Client (https://min.io/docs/minio/linux/reference/minio-mc.html)
+  # to bootstrap a MinIO Bucket as MinIO doesn't come prepopulated with one, and
+  # Bufstream doesn't create one itself.
+  mc:
```

Comment:
There *must* be a simpler way to do this I think, right? Like a CLI tool or something we can tell users to call to create the bucket? I would prefer that over adding this to the docker-compose file - I want users to see the docker-compose file and realize how simple it is to run Bufstream